tests/components/open_meteo/conftest.py

```diff
@@ -0,0 +1,49 @@
+"""Fixtures for the Open-Meteo integration tests."""
+from __future__ import annotations
+
+from collections.abc import Generator
+from unittest.mock import MagicMock, patch
+
+from open_meteo import Forecast
+import pytest
+
+from homeassistant.components.open_meteo.const import DOMAIN
+from homeassistant.const import CONF_ZONE
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        title="Home",
+        domain=DOMAIN,
+        data={CONF_ZONE: "zone.home"},
+        unique_id="zone.home",
+    )
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[None, None, None]:
+    """Mock setting up a config entry."""
+    with patch(
+        "homeassistant.components.open_meteo.async_setup_entry", return_value=True
+    ):
+        yield
+
+
+@pytest.fixture
+def mock_open_meteo(request: pytest.FixtureRequest) -> Generator[None, MagicMock, None]:
+    """Return a mocked Open-Meteo client."""
+    fixture: str = "open_meteo/forecast.json"
+    if hasattr(request, "param") and request.param:
+        fixture = request.param
+
+    forecast = Forecast.parse_raw(load_fixture(fixture))
```

Comment:
`load_fixture` modern approach is passing in the integration name as 2nd arg: `load_fixture('forecast.json', DOMAIN)`

Reply:
I added support for `DOMAIN/FIXTURE_NAME` as a fallback https://github.com/home-assistant/core/blob/dev/tests/common.py#L405-L406